libraries/outputs.rb

```diff
@@ -24,6 +24,35 @@ def self.configure_outputs(node) # rubocop:disable Metrics/PerceivedComplexity,
         else
           [CernerSplunk.my_cluster(node)]
         end.each do |(cluster, bag)|
+          if bag['indexer_discovery'] == true
+            Chef::Log.warn "Found ['receivers'] configured in cluster #{cluster} when ['indexer_discovery'] is set to true." if bag['receivers']
```

Comment:
minor, the wording here could be a bit more clear.  Something along the lines of "Configured ['receivers'] in cluster #{cluster} will be ignored since ['indexer_discovery'] is set to true."

Reply:
b88a9564d9fad8296edff5790f0db68ee493299e